core/multigroup/multigroup_types.py

```diff
@@ -30,13 +30,15 @@ def create_multigroup(bm, faces, prop):
     """
 
     # Prevent error when there are no components
-    if len(str(prop.components)) == 0:
+    if len(prop.components) == 0:
         popup_message("No components are chosen", "No Components Error")
+        prop.components = "dw"
```

Comment:
if there are no components, I'd prefer that we do nothing but return after showing the popup error.